integration_tests/tests/end_to_end.rs

```diff
@@ -256,7 +246,6 @@ async fn e2e_test_wrapper<T: Future<Output = ()>>(test: T) {
             output = lard_api::run(api_pool) => output,
             _ = init_shutdown_rx1.recv() => {
                 api_shutdown_tx.send(()).unwrap();
```

Comment:
should the semicolon now be dropped from this line?

Reply:
No, we don't need to return the result of send, the unit type `()` is returned automatically.